README.md

```diff
@@ -17,26 +21,25 @@ The cookbook is structured as follows:
 
 ## Getting Started
 
-The easiest way to use the COSIMA Cookbook is through NCI's HPC systems (either VDI or Gadi). The cookbook is preinstalled in the latest `conda/analysis3` environment.
+The easiest way to use the COSIMA Cookbook is through NCI's HPC systems (either via the Australian Research Environment (ARE), Open OnDemand (OOD), or on Gadi). The cookbook is preinstalled in the latest `conda/analysis3` environment.
```

Comment:
missed this line

Reply:
Should we ditch OOD?

Reply:
I believe OOD is being deprecated, but no idea of the timeframe.

Reply:
nuked OOD